src/components/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+const divStyle = {
+  width: '500px',
+  margin: 'auto',
+};
+
+function App() {
+  return (
+    <div style={divStyle}>
+      <Display result="0" />
```

Comment:
- [ ] [MANDATORY]  This is correct but it would be better to initialize default value inside `Display` component. like so...
```Javascript
    Display.defaultProps = {
        result: 0,
    }
```